docs/v1/P4Runtime-Spec.mdk

```diff
@@ -686,10 +686,11 @@ follows:
   for each pair of (`device_id`, `role_id`) at any point of time.
 
 * The streaming channel between the controller and the server defines the
-  liveness of the controller session. The controller is considered "offline" or
-  "dead" as soon as its corresponding stream channel to the switch is
-  broken, in which case the P4Runtime server quickly sets one of the slave
-  controllers with the highest `election_id` as master.
+  liveness of the controller session. The controller is considered “offline” or
+  “dead” as soon as its corresponding stream channel to the switch is broken, in
+  which case the P4Runtime server will be without a master, until one of the
+  slave controllers increases its election_id to a value higher than anything
```

Comment:
Question that I am not sure about: Is it important that the new controller must have an election_id _strictly greater_ than any election_id value that the server has previously considered the master?

Or is it acceptable that it be _at least_ the previous master's election_id?

Reply:
It's important that the election_id can be equal. Consider the case where the master disconnects, and then reconnects immediately. It would still have the same election id, and should be elected master again.

Reply:
OK, that sounds like a detail good to include in any formal analysis done on this, as it sounds racy to me.